### PR TITLE
Update arm template

### PR DESCRIPTION
**Pull request by maririos**
- Removing Shift Left reference,
- Adding clarity about PRs to publish a previously private PR
- Update stale links

### CODE REVIEW

**Review by konrad-jamrozik — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md [hunk 1]
```diff
@@ -46,8 +42,8 @@ Otherwise your PR may be subject to ARM review requirements. Complete the follow
 ### Breaking Change Review Checklist
 If you have any breaking changes as defined in the [Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy/), request approval from the Breaking Change Review Board.
 
-**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).
+**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Additional details on the process and office hours are on the [Breaking Change Wiki](http://aka.ms/aprwiki).
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:45]: This looks like the wrong link.

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md [hunk 2]
```diff
@@ -37,7 +33,7 @@ Otherwise your PR may be subject to ARM review requirements. Complete the follow
   - Adding a new service
   - Adding new API(s)
   - Adding a new API version
-    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).
+    -[ ] To review changes efficiently, ensure you copy the existing version into the new directory structure for first commit and then push new changes, including version updates, in separate commits. You can use OpenAPIHub to initialize the PR for adding a new version. For more details refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version). Note that this doesn't apply if you are trying to merge a PR that was previously in the private repository.
```
Comment (weshaggard) [inline .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:36]: Do we have an aka link for this doc?

Comment (maririos) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:36]: Not currently. I also don't expect to keep this page once we move to Eng hub as it is a copy paste of OpenAPI Hub

File: .github/PULL_REQUEST_TEMPLATE/control_plane_template.md (hunk 1 above)
Comment (maririos) [reply .github/PULL_REQUEST_TEMPLATE/control_plane_template.md:45]: Thanks. I will fix it

**Pull request merged**